xy/rdp_simplify.go

```diff
@@ -0,0 +1,140 @@
+package xy
+
+import (
+	"math"
+)
+
+const stride = 2
```

Comment:
This should be a parameter, not a fixed constant.

Reply:
Hi, thanks for reviewing.

I had a geom.Layout parameter, as you can see below all using stride variables. But the package already implies it's 2D geometry. What do you think?